portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,17 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+function randomMessage(){
+  console.log('Inside randomMessage function.. ');
+  // const msgContainer =  document.getElementById('msg-container');
+  // msgContainer.innerText = "Ur Special!!";
```

Comment:
Should these be removed?

Reply:
I'm keeping this.. to remember the old way of doing it.. :sweat_smile:

Reply:
Do you want to add a line on top of these (on 39) stating the same. So others looking at this in later patches don't confuse this with dead/unused code and understand the reasons for it better.

Reply:
there are lot of changing in the tutorials.. I'm removing the older way..
new ways are much better and cooler.